canonical-web-and-design/ubuntu.com
# Add count to closed accordion

## Done

- Add counter to accordion tab when the accordion is closed
- Move javascript functionality into it's own file

## QA

- View page at: https://ubuntu-com-10125.demos.haus/certified?q=
- Select filters on the LHS and close the accordion - check the chip is visible on each section with the correct number of filters applied. 
- Check when there are no filters applied the chip isn't visible. 


## Issue / Card

Fixes [#4250](https://github.com/canonical-web-and-design/web-squad/issues/4250)

## Screenshots

![Screenshot 2021-08-04 at 11 20 40](https://user-images.githubusercontent.com/58959073/128165128-aed149d9-002b-4852-a71b-dd31be8ec4e6.png)
